### ISSUE TITLE
Cannot covert a Charles proxy HAR file into a script

### ISSUE THREAD

**Issue by Ardesco** [loadimpact/k6#793]
Looks like the HAR convertor does not work with har files saved from Charles proxy.

Added an example HTTP Archive and the script created by running the following command (both files in the attached zip):

```
k6 convert example.har -O example.js
```

[example.zip](https://github.com/loadimpact/k6/files/2437399/example.zip)

**Comment by na--** [comment]
Confirmed, this is a bug - seems like the k6 HAR converter [assumes](https://github.com/loadimpact/k6/blob/c665a73e561bcef243b40207640d8099d392f488/converter/har/converter.go#L133) that there's a `pages` key in the HAR file. This makes sense for HAR files recorded in browsers, but proxies would have no idea what to put there so they don't include it. And regardless, the [spec](https://dvcs.w3.org/hg/webperf/raw-file/tip/specs/HAR/Overview.html) says that `pages` is optional, so we shouldn't depend on it...

Until we fix this (probably together with the rest of the [several](https://github.com/loadimpact/k6/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+har) HAR converter improvement we want to make and issues we have to fix), you can work around the problem by manually editing the `.har` file and inserting a dummy page with `""` for ID, like this:

```js
{
  "log": {
    "version": "1.2",
    "creator": {
      "name": "Charles Proxy",
      "version": "4.2.7"
    },
    "pages": [
      {
        "id": "",
        "title": "dummy"
      }
    ],
    "entries": [
      /* ... */
    ]
  }
}
```

**Comment by Ardesco** [comment]
Workaround works perfectly, thanks :)

**Comment by na--** [comment]
Received another issue about this (non-Charles Proxy related), so I think we should fix it before 1.0. The quick fix should be easy enough, we can just officially use the minor hack I proposed as a workaround - inserting a placeholder element in `pages` if it's empty... We're going to majorly refactor (for [this](https://github.com/loadimpact/k6/issues/762) among other reasons) the HAR converter post 1.0, so we'll properly deal with it when that happens, but the quick fix should suffice until then.
